roles/openafs_client/tasks/CentOS.yaml

```diff
@@ -1,7 +1,7 @@
 ---
-- name: Disable selinux
-  selinux:
-    state: disabled
+#- name: Disable selinux
```

Comment:
Hi, I'd prefer if we did not comment out code, just remove it. Thanks.

Reply:
Done.  Removed the commented blocks in latest commit